champions/src/main/java/me/mykindos/betterpvp/champions/listeners/ArrowListener.java

```diff
@@ -48,8 +49,13 @@ public void onShootBow(EntityShootBowEvent event) {
         if (event.getProjectile() instanceof Arrow arrow) {
             if (event.getEntity() instanceof Player player) {
                 arrow.setMetadata("ShotWith", new FixedMetadataValue(champions, player.getInventory().getItemInMainHand().getType().name()));
+
+
+                Vector direction = player.getLocation().getDirection().normalize();
+                arrow.setVelocity(direction.multiply(event.getForce()));
+
```

Comment:
Why?

Reply:
This removes the randomness from arrows, which from my testing was why arrows In alpha sometimes felt strange

Reply:
AFAIK it's also because of the arrow spawn location. Mineplex made it spawn at your bow, whereas vanilla adds the randomness I think.  

In any case, you're right about the velocity.
![image](https://github.com/user-attachments/assets/33a1bd89-90a7-48a6-9dcd-ba4ec583f42d)